iac/main.tf

```diff
@@ -15,7 +15,15 @@ terraform {
 
 # Configure the AWS Provider
 provider "aws" {
-  region = "us-east-1"
+  region = var.aws_region
```

Comment:
Defining `default_tags` at the provider level ensures that ALL resources created by this provider inherit these tags. This is excellent for cost allocation, ownership tracking, and resource management.